src/auro-flight-main.js

```diff
@@ -42,11 +46,55 @@ class AuroFlightMain extends LitElement {
     `;
   }
 
+  connectedCallback() {
+    super.connectedCallback();
+
+    /**
+     * Time template object used by convertTime() method.
+     */
+    this.timeTemplate = {
+      hour: "2-digit",
+      minute: "2-digit",
+      timeZone: this.timeZone
+    };
+
+    this.template = {};
+  }
+
+  /**
+   * @private
+   * @param {string} time - UTC time.
+   * @returns Localized time based from UTC string.
+   */
+  convertTime(time) {
+    const slicedTime = time.slice(0, -6);
+    const newTime = new Date(slicedTime);
+    const localizedTime = newTime.toLocaleString('en-us', this.timeTemplate).replace(/^0+/u, '');
+
+    return localizedTime;
+  }
+
   // function that renders the HTML and CSS into  the scope of the component
   render() {
     return html`
+        <script type="application/ld+json">
+          {
+            "@context": "https://schema.org/",
+            "@type": "Flight",
+            "departureTime": "${this.departureTime}",
+            "arrivalTime": "${this.arrivalTime}",
+            "estimatedFlightDuration": "${this.duration}",
+            "name": "Flight(s) ${this.flights}",
+            "arrivalAirport": "${this.arrivalStation}",
+            "departureAirport": "${this.departureStation}",
+            "description": "Departs from ${this.departureStation} at ${this.convertTime(this.departureTime)}, arrives ${this.arrivalStation} at ${this.convertTime(this.arrivalTime)}"
```

Comment:
Nice to have: read out intermediate stopovers.

Reply:
example - a nonstop from SEAANC will read the same as a milk run flight with 5 stops in between SEA and ANC.

Reply:
New issue created

- https://github.com/AlaskaAirlines/auro-flight/issues/34

I will consult with you later as to the scope of that feature.